pkg/vmtest/integration.go

```diff
@@ -339,6 +339,7 @@ func CreateTestInitramfs(dontSetEnv bool, o uroot.Opts, uinit, outputFile string
 		env.CgoEnabled = false
 		env.GOARCH = TestArch()
 		o.Env = env
+		o.UrootSource = filepath.Join(env.GOPATH, "src/github.com/u-root/u-root")
```

Comment:
That's dangerous, not every u-root user uses Gopath

Reply:
Changed so it will evaluate the UROOT_SOURCE env and bail out with a message if that is not set